### DISCUSSION TITLE
Any idea to bypass AWS WAF CAPTCHA

### DISCUSSION THREAD

**Discussion by ByteGui** [seleniumbase/SeleniumBase#3032] [General]
Hi,

I’m currently facing an issue with AWS WAF CAPTCHA while using a particular website. As long as I keep a low profile and don’t use the sb.uc_open() method (or the sb.uc_open_with_reconnect() method) more than once every 7 seconds, everything works smoothly. However, if I try to increase the frequency, AWS WAF CAPTCHA flags my activity.

It seems this issue is directly related to my IP address, as replicating this behavior manually in my browser results in the same CAPTCHA challenge at similar intervals.

I’ve considered using proxies and have already set up a Python script that utilizes OpenVPN to connect to this challenging website through 12,775 NordVPN servers, one at a time. The script has been running for 18 hours, and while half of the servers have been tested, none have resulted in successful connections (I’ve verified the integrity of this script on other websites beforehand).

Given the situation, two potential solutions come to mind:

	1.	Bypassing AWS WAF CAPTCHA:

![image](https://github.com/user-attachments/assets/7dc50e4a-e8d7-4b16-be7e-f86a349dbc00)

	2.	Using reliable proxies: However, finding dependable ones is quite challenging.

Which approach do you think would be more effective? Any advice or tips would be greatly appreciated.

Thanks in advance!

**Comment by immortal-stem** [comment]
Have you tried using residential proxies??

**Comment by ByteGui** [comment]
I've resolved the problem balancing delay between each connection more randomly

**Reply by cuong02n** [reply]
what is the strategy of your random?
